int8 elbowVel
int8 shoulderVel
int8 baseVel
